syntax = "proto3";

package envoymobile.extensions.filters.http.test_kv_store;

import "buf/validate/validate.proto";

message TestKeyValueStore {
  string kv_store_name = 1 [(buf.validate.field).string.min_len = 1];
  string test_key = 2 [(buf.validate.field).string.min_len = 1];
  string test_value = 3 [(buf.validate.field).string.min_len = 1];
}
